src/pudl/metadata/fields.py

```diff
@@ -4012,6 +4032,63 @@
         "type": "string",
         "description": "Label describing types of revenues.",
     },
+    "revenue_requirement_category": {
+        "type": "string",
+        "description": (
+            "A category of revenue requirement components. "
+            "These categories were mannually compiled by RMI utility accounting experts "
+            "based on the xbrl_factoid and sometimes vary based on the utility_type or "
+            "plant_function as well."
+        ),
+        "constraints": {
+            "enum": [
+                "depreciation_amortization_depletion",
+                "depreciation_arc",
+                "fuel",
+                "investment_tax_credit",
+                "maintenance",
+                "non_fuel_operation",
+                "other",
+                "purchased_power",
+                "regulatory_debits_credits",
+                "taxes",
+            ]
+        },
+    },
+    "revenue_requirement_technology": {
+        "type": "string",
+        "description": (
+            "The technology type associated with components of a utility's "
+            "revenue requirement. "
+            "These categories were mannually compiled by RMI utility accounting experts "
+            "based on the xbrl_factoid and sometimes vary based on the utility_type or "
+            "plant_function as well."
```

Comment:
Maybe call out the `revenue_requirement_category` column by name instead of referencing it by saying "components of a utility's revenue requirement"